ampl-converter/src/main/java/com/powsybl/ampl/converter/version/AmplExportVersion.java

```diff
@@ -24,7 +24,8 @@
  */
 public enum AmplExportVersion {
 
-    V1_0("1.0", BasicAmplExporter.getFactory());
+    V1_0("1.0", BasicAmplExporter::new),
```

Comment:
The default version seems to be 1.0. It is not the classical choice, usually we choose the highest one.

Reply:
The default version is now the extended version of the exporter. Thank you for the comment.